scripts/protoc-swagger-openapi-gen.sh

```diff
@@ -27,7 +27,7 @@ if [ -d $temp_dir ]; then
   mv ./$temp_dir ./vendor
 fi
 # XXX We will get rid of erc20 proto files on ethermint, so I remove erc20 files on this list temporary.
```

Comment:
The comments below also seem unnecessary now.

Reply:
[f8500dc](https://github.com/xpladev/xpla/pull/174/commits/f8500dcdfb2d09cc429e2d1fdab75b61a859a9d0)